htmd/queues/localqueue.py

```diff
@@ -362,20 +365,35 @@ def ngpu(self, value):
 
     @property
     def memory(self):
-        return self.__dict__['memory']
+        return self._getmemory()
```

Comment:
If we have this, a user can never setup the memory to something he wants. Right?

Reply:
I haven't found a nice way to implement this default.

Reply:
Isn't the `self._arg('memory', 'int', 'The amount of RAM memory available', self._getmemory(),` working?

Reply:
I mean, the constructor itself should not have any default. Only the initialized object should have a default.

Reply:
yeah do it like j3damas suggests. The property should return the value set by the user, not the total available memory. If you want to get all available memory just call self._getmemory()

Reply:
I don't understand what was the problem. I added now defaults to LocalCPUQueue and they work swell. https://github.com/Acellera/htmd/commit/2783af22d97dacf2e31d591a58f50dfbc9cbb5fd